modules/brainbrowser/js/brainbrowser.loris.js

```diff
@@ -1,577 +1,579 @@
 /*global window, BrainBrowser, unescape, $*/
-function getQueryVariable(variable) {
```

Comment:
Indentation. It had to go in a $(document).ready() wrapper because it needs to be executed after the DOM is loaded (before, it was in a script tag at the bottom of the template where the DOM is already loaded, implementing getJSDependencies() moves it to the top). I'm not sure why GitHub diffs it so weirdly.

I think there may have been 1 or 2 calls to AjaxHelper which were changed to pretty URLs, but 90% of it is indentation..